ldk/javascript/src/whisper/mapper.ts

```diff
@@ -218,35 +199,20 @@ export function mapToInternalComponent(
   switch (component.type) {
     case WhisperComponentType.CollapseBox:
       // eslint-disable-next-line no-case-declarations
-      const { onClick } = component;
-      if (onClick) {
-        return {
-          label: component.label,
-          open: component.open,
-          key: component.key,
-          children: throwForDuplicateKeys(
-            component.children.map((childComponent) =>
-              mapToInternalChildComponent(childComponent, stateMap),
-            ),
-          ),
-          type: WhisperComponentType.CollapseBox,
-          onClick: (error: Error, param: boolean, whisper: OliveHelps.Whisper) => {
-            onClick(error, param, mapToExternalWhisper(whisper, stateMap));
-          },
-        } as OliveHelps.CollapseBox;
-      }
+      const onClick = 'onClick' in component ? component.onClick : null;
```

Comment:
Simplified these because sidekick doesn't care if the property is undefined when it receives it.

Reply:
cool